### PR TITLE
test: Setup jest testing

### PR DESCRIPTION
**Pull request by parcpaes**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "javascriptprojects",
+  "version": "1.0.0",
+  "description": "html,css,js practice\r # Web design",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (parcpaes) [inline package.json:7]: Can you set Jest's configuration to the test property?

Comment (parcpaes) [reply package.json:7]: Sure, I will fix

File: package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
```
Comment (parcpaes) [inline package.json:7]: Can you Set up Jest configuration

Comment (paes-hermione) [reply package.json:7]: Sure, I'll do

File: package.json (hunk 1 above)
Comment (paes-hermione) [inline package.json:7]: Can you set up Jest configuration in the test property?

Comment (parcpaes) [reply package.json:7]: Sure, I'll do

Comment (parcpaes) [reply package.json]: Sure, I'll do it

File: package.json (hunk 2 above)
Comment (paes-hermione) [inline package.json]: Can you set Jest in the test property?

Comment (parcpaes) [reply package.json]: Sure, I'll do it

**Review by paes-hermione — CHANGES_REQUESTED** [review]
Please, change the configuration on the review

**Review by paes-hermione — APPROVED** [review]
Well Done

**Pull request merged**